File: 241023/학점 계산기/credit-calculator.py
n = int(input())
arr = list(map(float ,input().split()))

sums = 0;
for i in arr:
    sums+= i

print(f'{sums/n:.1f}')
if (sums/n >= 4.0):
    print('Perfect')
elif (sums/n >= 3.0):
    print('Good')
else:
    print('Poor')

File: 250623/학점 계산기/credit-calculator.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int n;
    cin >> n;
    double arr[5];
    double sums=0;
    cout << fixed;
    cout.precision(1);
    for (int i=0; i<n; i++) {
        cin >>arr[i];
        sums+=arr[i];
    }
    double res = sums/n;
    cout << res << "\n";
    if (res>=4.0) {
        cout << "Perfect";
    }
    else if (res >=3.0) {
        cout << "Good";
    }
    else {
        cout << "Poor";
    }
    return 0;
}